src/o3_component.cpp

```diff
@@ -62,6 +62,11 @@ void OzoneComponent::init(Core *coreptr) {
   core->registerInput(D_EMISSIONS_CO, getComponentName());
```

Comment:
@bpbond before I wrap up the changes for this PR I do have a question, looking at the [src/o3_component.cpp](https://github.com/JGCRI/hector/blob/a61d64af3b9cc6d0d63cd661ee0164067a8f0165/src/o3_component.cpp#L60) 

```
  // Register the emissions we accept (note that OH component also
  // accepts these.  That's ok
  core->registerInput(D_EMISSIONS_CO, getComponentName());
  core->registerInput(D_EMISSIONS_NMVOC, getComponentName());
  core->registerInput(D_EMISSIONS_NOX, getComponentName());
  core->registerInput(D_EMISSIONS_H2, getComponentName());
```
which makes me think that we can also read in the H2 emissions a second time here? Is that how we want to do this? Or should these emissions be passed as a dependency from the OH component?

Reply:
@kdorheim No, I agree, it seems cleanest to have it as a direct input here too. 

That surprises me a little! I didn't realize (or forgot) the model could route inputs to multiple components.

Reply:
So leave it as is?

Reply:
That's my vote, yes